src/compiler/checker.ts

```diff
@@ -11336,6 +11336,10 @@ namespace ts {
             return !!getPropertyOfType(type, "0" as __String);
         }
 
+        function isNotUnitTypeOrNever(type: Type): boolean {

```

Comment:
Nit: isNeitherUnitTypeNorNever

Reply:
But I don't use the neither/nor operator in the body. 👅